cypress/support/commands/recaptcha.ts

```diff
@@ -3,22 +3,46 @@ declare global {
 	namespace Cypress {
 		interface Chainable {
 			interceptRecaptcha: typeof interceptRecaptcha;
+			failRecaptchaRequest: typeof failRecaptchaRequest;
+			reEnableRecaptchaRequest: typeof reEnableRecaptchaRequest;
 		}
 	}
 }
 
 /**
  * Simulate an error with recaptcha so we can test the error messaging and user behaviour
  */
-export const interceptRecaptcha = (times = 1) => {
+export const interceptRecaptcha = (times: number | 'indefinate' = 1) => {
 	cy.intercept(
 		{
 			method: 'POST',
 			url: 'https://www.google.com/recaptcha/api2/**',
-			times,
+			...(typeof times === 'number' && { times }),
 		},
 		{
 			statusCode: 500,
 		},
 	);
 };
```

Comment:
This was part of an investigation into the recaptcha request interceptions. Although this new functionality isn't used by the any of the tests, I thought it might be useful to someone in the future so I've left it in here. Basically the test will default to intercept the request one time, you can override that by passing a number to reflect the number of times you'd like the request to be intercepted, or pass through the string 'indefinate' which will intercept any number of requests.